pkgs/development/libraries/gstreamer/rs/default.nix

```diff
@@ -139,30 +139,37 @@ stdenv.mkDerivation rec {
     owner = "gstreamer";
     repo = "gst-plugins-rs";
     rev = version;
-    hash = "sha256-UxmfyqbQwkQjwHiARRpFJiGsrsNjv6V129lIHPk7gRk=";
+    hash = "sha256-pUvcPazg6/SsD7sqML2tb+ZVWDqO1MRIcj3CQioePZY=";
     # TODO: temporary workaround for case-insensitivity problems with color-name crate - https://github.com/annymosse/color-name/pull/2
-    nativeBuildInputs = [ yq moreutils ];
     postFetch = ''
-      tomlq --toml-output '.package |= map(if .name == "color-name"
-        then (.source = "git+https://github.com/lilyinstarlight/color-name#cac0ed5b7d2e0682c08c9bfd13089d5494e81b9a" | del(.checksum))
-        else .
-      end)' $out/Cargo.lock | sponge $out/Cargo.lock
+      sedSearch="$(cat <<\EOF | sed -ze 's/\n/\\n/g'
+      \[\[package\]\]
+      name = "color-name"
+      version = "\([^"\n]*\)"
+      source = "registry+https://github.com/rust-lang/crates.io-index"
+      checksum = "[^"\n]*"
+      EOF
+      )"
+      sedReplace="$(cat <<\EOF | sed -ze 's/\n/\\n/g'
+      [[package]]
+      name = "color-name"
+      version = "\1"
+      source = "git+https://github.com/lilyinstarlight/color-name#cac0ed5b7d2e0682c08c9bfd13089d5494e81b9a"
+      EOF
+      )"
+      sed -i -ze "s|$sedSearch|$sedReplace|g" $out/Cargo.lock
```

Comment:
Can't we patch that? Potentially updating this sounds like a pain.

Reply:
The crate hasn't been updated in ages and I want to get upstream off of it altogether

Doing it as a patch is actually more of a pain for updating because it would cause surrounding diff conflicts when this one depependecy doesn't actually change

Reply:
It's on my todo list to get upstream on another crate, but unfortunately all of the crates I surveyed wouldn't quite have a one-to-one mapping with the color names in this crate. I'll probably just open an issue asking what they would prefer to do about it

I've also considered forking it and uploading my own crate to crates.io, but that would probably seem more suspicious to the gst-plugins-rs project and I was really hoping the original maintainer would just respond to https://github.com/annymosse/color-name/pull/2 and avoid these contingencies...

Reply:
Then we just keep it for now and make it better later